### PR TITLE
Update tidb-configuration-file.md

### PR DESCRIPTION
**Pull request by TomShawn**
### What is changed, added or deleted? (Required)

Documentation for pingcap/tidb#21552

### Which TiDB version(s) do your changes apply to? (Required)

- [x] master (the latest development version)
- [x] v5.0 (TiDB 5.0 versions)
- [ ] v4.0 (TiDB 4.0 versions)
- [ ] v3.1 (TiDB 3.1 versions)
- [ ] v3.0 (TiDB 3.0 versions)
- [ ] v2.1 (TiDB 2.1 versions)

### What is the related PR or file link(s)?

- This PR is translated from: https://github.com/pingcap/docs/pull/5169
- Other reference link(s):

### Do your changes match any of the following descriptions?

- [ ] Delete files
- [ ] Change aliases
- [ ] Need modification after applied to another branch 
- [ ] Might cause conflicts after applied to another branch

### CODE REVIEW

File: tidb-configuration-file.md [hunk 1]
```diff
@@ -157,6 +157,12 @@ TiDB 配置文件比命令行参数支持更多的选项。你可以在 [config/
 + 默认值：true
 + 如果所有 TiDB 实例上该选项都设置为 `false`，那么将完全禁用 TiDB 遥测功能，且忽略 [`tidb_enable_telemetry`](/system-variables.md#tidb_enable_telemetry-从-v402-版本开始引入) 系统变量。参阅[遥测](/telemetry.md)了解该功能详情。
 
+### `enable-tcp4-only` <span class="version-mark">从 v5.0.0 版本开始引入</span>
+
++ 控制是否只监听 TCP4。
++ 默认值：false
++ 如果 TiDB 和 LVS 一起用于均衡负载，可开启此配置项。这是因为 TCP4 协议能正确解析 TCP 头部信息中的真实 IP。详情参阅 [LVS/toa](https://github.com/alibaba/LVS/tree/master/kernel/net/toa)。
```
Comment (tangenta) [inline tidb-configuration-file.md:164]:
```suggestion
+ 当使用 LVS 为 TiDB 做负载均衡时，可开启此配置项。这是因为 TCP4 协议能正确解析 TCP 头部信息中的真实 IP。详情参阅 [LVS/toa](https://github.com/alibaba/LVS/tree/master/kernel/net/toa)。
```

File: tidb-configuration-file.md [hunk 2]
```diff
@@ -157,6 +157,12 @@ TiDB 配置文件比命令行参数支持更多的选项。你可以在 [config/
 + 默认值：true
 + 如果所有 TiDB 实例上该选项都设置为 `false`，那么将完全禁用 TiDB 遥测功能，且忽略 [`tidb_enable_telemetry`](/system-variables.md#tidb_enable_telemetry-从-v402-版本开始引入) 系统变量。参阅[遥测](/telemetry.md)了解该功能详情。
 
+### `enable-tcp4-only` <span class="version-mark">从 v5.0.0 版本开始引入</span>
+
++ 控制是否只监听 TCP4。
++ 默认值：false
++ 当使用 LVS 为 TiDB 做负载均衡时，可开启此配置项。这是因为 TCP4 协议能正确解析 TCP 头部信息中的真实 IP。详情参阅 [LVS/toa](https://github.com/alibaba/LVS/tree/master/kernel/net/toa)。
```
Comment (tangenta) [inline tidb-configuration-file.md:164]:
```suggestion
+ 当使用 LVS 为 TiDB 做负载均衡时，需开启此配置项。这是因为 TCP4 协议才能正确解析 TCP 头部信息中的真实客户端 IP。详情参阅 [LVS/toa](https://github.com/alibaba/LVS/tree/master/kernel/net/toa)。
```

**Review by tangenta — COMMENTED** [review]
/lgtm

File: tidb-configuration-file.md [hunk 3]
```diff
@@ -157,6 +157,12 @@ TiDB 配置文件比命令行参数支持更多的选项。你可以在 [config/
 + 默认值：true
 + 如果所有 TiDB 实例上该选项都设置为 `false`，那么将完全禁用 TiDB 遥测功能，且忽略 [`tidb_enable_telemetry`](/system-variables.md#tidb_enable_telemetry-从-v402-版本开始引入) 系统变量。参阅[遥测](/telemetry.md)了解该功能详情。
 
+### `enable-tcp4-only` <span class="version-mark">从 v5.0.0 版本开始引入</span>
```
Comment (TomShawn) [inline tidb-configuration-file.md:160]:
```suggestion
### `enable-tcp4-only` <span class="version-mark">从 v5.0 版本开始引入</span>
```

File: tidb-configuration-file.md (hunk 2 above)
Comment (qiancai) [inline tidb-configuration-file.md]:
```suggestion
+ 当使用 LVS 为 TiDB 做负载均衡时，可开启此配置项。这是因为 [LVS 的 TOA 模块](https://github.com/alibaba/LVS/tree/master/kernel/net/toa)可以使 TCP4 协议从 TCP 头部信息中解析出客户端的真实 IP。详情参阅 [LVS/toa。
```

File: tidb-configuration-file.md [hunk 4]
```diff
@@ -157,6 +157,12 @@ TiDB 配置文件比命令行参数支持更多的选项。你可以在 [config/
 + 默认值：true
 + 如果所有 TiDB 实例上该选项都设置为 `false`，那么将完全禁用 TiDB 遥测功能，且忽略 [`tidb_enable_telemetry`](/system-variables.md#tidb_enable_telemetry-从-v402-版本开始引入) 系统变量。参阅[遥测](/telemetry.md)了解该功能详情。
 
+### `enable-tcp4-only` <span class="version-mark">从 v5.0 版本开始引入</span>
+
++ 控制是否只监听 TCP4。
++ 默认值：false
++ 当使用 LVS 为 TiDB 做负载均衡时，需开启此配置项。这是因为 TCP4 协议才能正确解析 TCP 头部信息中的真实客户端 IP。详情参阅 [LVS/toa](https://github.com/alibaba/LVS/tree/master/kernel/net/toa)。
```
Comment (TomShawn) [inline tidb-configuration-file.md:164]:
```suggestion
+ 当使用 LVS 为 TiDB 做负载均衡时，可开启此配置项。这是因为 [LVS 的 TOA 模块](https://github.com/alibaba/LVS/tree/master/kernel/net/toa)可以用 TCP4 协议从 TCP 头部信息中解析出客户端的真实 IP。
```

File: tidb-configuration-file.md [hunk 5]
```diff
@@ -157,6 +157,12 @@ TiDB 配置文件比命令行参数支持更多的选项。你可以在 [config/
 + 默认值：true
 + 如果所有 TiDB 实例上该选项都设置为 `false`，那么将完全禁用 TiDB 遥测功能，且忽略 [`tidb_enable_telemetry`](/system-variables.md#tidb_enable_telemetry-从-v402-版本开始引入) 系统变量。参阅[遥测](/telemetry.md)了解该功能详情。
 
+### `enable-tcp4-only` <span class="version-mark">从 v5.0 版本开始引入</span>
+
++ 控制是否只监听 TCP4。
++ 默认值：false
++ 当使用 LVS 为 TiDB 做负载均衡时，可开启此配置项。这是因为 [LVS 的 TOA 模块](https://github.com/alibaba/LVS/tree/master/kernel/net/toa)可以用 TCP4 协议从 TCP 头部信息中解析出客户端的真实 IP。
```
Comment (TomShawn) [inline tidb-configuration-file.md:164]:
```suggestion
+ 当使用 LVS 为 TiDB 做负载均衡时，可开启此配置项。这是因为 [LVS 的 TOA 模块](https://github.com/alibaba/LVS/tree/master/kernel/net/toa)可以通过 TCP4 协议从 TCP 头部信息中解析出客户端的真实 IP。
```

**Pull request merged**